[IMP] accounting: withholding tax (#6709)
odoo/documentation: content/applications/finance/accounting/taxes/retention.rst

Withholding taxes
=================

A **withholding tax**, or retention tax, is a government requirement for the payer of a
customer invoice to withhold or deduct tax from the payment and pay that tax to the government. In
most jurisdictions, withholding tax applies to employment income.

Usually, the tax is added to the subtotal to give the total amount to pay. In the case of
withholding taxes, they are deducted from the amount to pay as the customer pays the tax.

Configuration
=============

> In Odoo, a withholding tax is defined by creating a negative tax. To create a retention of 10%, go

Reviewer: Is 10% a standard? If not, I would maybe generalize this sentence, and then add the 10% as an example, with the 2 screenshots. This would also avoid having two screenshots with a sentence sandwiched (and a little bit lost) in between, then another one sandwiched between the screenshot and the tip.  Let's discuss it :)

to :menuselection:`Configuration --> Taxes` and in the :guilabel:`Amount` field insert the amount as
a negative quantity.

.. image:: retention/negative-amount.png
   :alt:   negative tax amount in field

Go to the :menuselection:`Advanced Options` tab and set a specific Retention :guilabel:`Tax Group`.

.. image:: retention/tax-group.png
   :alt: tax group for retention tax.

> Once the tax is defined, it can be used for products, sales orders and invoices.

Reviewer: ```suggestion
Once the tax is defined, it can be used for products, sales orders, and invoices.
```

.. tip::
    If the retention is a percentage of a regular tax, create a :guilabel:`Tax` with a

>     :guilabel:`Tax Computation` as a :guilabel:`Group of Taxes` and set the two taxes in this group
>     (normal tax and retention).

Reviewer: I'm not super sure I understand this. Could we talk about it? In any case, i would change this:
```suggestion
    :guilabel:`Tax Computation` as a :guilabel:`Group of Taxes` and set both the regular and retention taxes in this group.
```

Retention taxes on invoices
===========================

Once the tax is created, it can be used on customer forms, sales orders, or customer invoices.
Several taxes on a single customer invoice line can be applied.

.. image:: retention/invoice-tax.png
   :alt: invoice lines with taxes

> .. seealso::

Reviewer: For some reason i don't see what's in the seealso, but i would remove the bullet list since there's only one thing in there.

Withholding taxes
=================

A **withholding tax**, or retention tax, is a government requirement for the payer of a
customer invoice to withhold or deduct tax from the payment and pay that tax to the government.
Usually, the tax is added to the subtotal to determine the total amount to pay. In the case of

> withholding taxes, they are deducted from the payment amount.

Reviewer 2: Suggestion to rephrase:
A withholding tax, also known as retention tax, mandates the payer of a customer invoice to deduct tax from the payment and remit it to the government. Typically, a tax is included in the subtotal to calculate the total amount paid, while withholding taxes are directly subtracted from the payment.

Configuration
=============

In Odoo, a withholding tax is defined by creating a negative tax. To create one, go
to :menuselection:`Accounting --> Configuration --> Taxes` and, in the :guilabel:`Amount` field,
enter a negative amount.

.. image:: retention/negative-amount.png

>    :alt:   negative tax amount in field

Reviewer 2: ```suggestion
   :alt: negative tax amount in field
```

Then, go to the :menuselection:`Advanced Options` tab and create a retention :guilabel:`Tax Group`.

.. image:: retention/tax-group.png
   :alt: tax group for retention tax.

.. tip::

>     If the retention is a percentage of a regular tax, create a :guilabel:`Tax` with a
>     :guilabel:`Tax Computation` as a :guilabel:`Group of Taxes`. Then, set both the regular tax and
>     the retention one in the :guilabel:`Definition` tab.

Reviewer 2: ```suggestion
   If the retention is a percentage of a regular tax, create a :guilabel:`Tax` with a
   :guilabel:`Tax Computation` as a :guilabel:`Group of Taxes`. Then, set both the regular tax and
   the retention one in the :guilabel:`Definition` tab.
```
